frontend/src/plugins/impl/CodeEditorPlugin.tsx

```diff
@@ -79,11 +85,6 @@ const CodeEditorComponent = (props: CodeEditorComponentProps) => {
       props.setValue(newValue);
     }
   });
-  // This is to sync the value from Python whenever the cell is updated
-  // as useState doesn't reinitialize on re-renders
-  useEffect(() => {
-    setLocalValue(props.value);
-  }, [props.value]);
```

Comment:
can we revert this or did this fix a bug? because last time we did a cleanup like this, it broke things unexpectedly